claim-check/code-samples/sample-3/azure-function/ClaimCheck.cs

```diff
@@ -29,23 +27,19 @@ public static class ClaimCheck
             // Getting connection information
             var serviceBusConnectionString = Environment.GetEnvironmentVariable("SERVICE_BUS_CONNECTION_STRING");
             var queueName = Environment.GetEnvironmentVariable("QUEUE_NAME");
-            var storageConnectionString = Environment.GetEnvironmentVariable("STORAGE_CONNECTION_STRING");
-
-            // Creating config for sending message
-            var config = new AzureStorageAttachmentConfiguration(storageConnectionString);
 
             // Creating and registering the sender using Service Bus Connection String and Queue Name
-            var sender = new MessageSender(serviceBusConnectionString, queueName);
-            sender.RegisterAzureStorageAttachmentPlugin(config);
-            
+            var client = new ServiceBusClient(serviceBusConnectionString);
+            ServiceBusSender sender = client.CreateSender(queueName);
+
```

Comment:
Why removing the attachment plugin?  I believe that is the point of this sample.  I know it is not available in T2 yet, but what are we proposing as the alternative?

Reply:
Sorry, this was a mistake caused by my carelessness! I only saw the annotation `// Creating config for sending message` , so misunderstood `config` as a parameter in sending message. After specific analysis, it seems cannot be replaced by other methods. If no other questions, I will close this PR. Apologies for the inconvenience.

Reply:
Ok, thanks